python/GangaLHCb/__init__.py

```diff
@@ -46,7 +46,8 @@
     configLHCb.addOption('SplitByFilesBackend', 'OfflineGangaDiracSplitter',
                      'Possible SplitByFiles backend algorithms to use to split jobs into subjobs,\
                       options are: GangaDiracSplitter, OfflineGangaDiracSplitter, splitInputDataBySize and splitInputData')
-
+    defaultLHCbDirac = 'v8r2p36'
```

Comment:
great I'll modify the lhcb_prepare script to pick up this version from the codebase rather than having it hard-coded in that repo :)